### PR TITLE
20 cohortprogress component

### LINKED ISSUE #20: Create CohortProgress component
When we display the dashboard of a user we want to also display their current Cohort Progress. So we will need to create a CohortProgress component to go within the dashboard component.

It's also a good idea to create some mock data to utilise within this component. Having the mock data set up in the same format as the wiki ensures that the real live data we send through later will be even more seamless to integrate.

_Think about whether this is likely to be functional or class based._ 

You may also need to ensure that this component has a client-side route set up too. 

Tasks:
- [x] Check the [wiki](https://github.com/kahikatea-2019/MOA/wiki) for relevant information.
- [x] Create component file. 
- [x] Create mock data which is in the same shape as the wiki and used within the component.
- [ ] Display CohortProgress component within the dashboard. 
- [ ] Write tests for this component.

You won't need to do any design just yet, just the basics to get this showing.

### PR DESCRIPTION
**Pull request by Bradleywh1**
completed the test for cohort progressbar mock data

close #20 cohortprogressbar component

### CODE REVIEW

**Review by bryceknz — DISMISSED** [review]
Looks good! My only request is that you modify the state in `ProgressBar` to use actual data from the database, now that our seeds are completed :-)

File: tests/client/components/progressBar.test.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+// Prevent <ProgressBar> from calling the API
+ProgressBar.prototype.componentDidMount = () => {}
+
+test('test runner is working', () => {
+  expect(true).toBeTruthy()
+})
+
+test('<ProgressBar />', () => {
+  const wrapper = shallow(<ProgressBar />)
+  expect(wrapper.find('p').text()).toBe('2341')
```
Comment (bryceknz) [inline tests/client/components/progressBar.test.js]: I couldn't figure out where `'2341'` should be coming from?

File: client/components/ProgressBar.jsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+       'status_id': 1,
+       'evidence': 'evidence',
+       'date_modified': '2019-02-24 22:55:20'
+     }
+   ]
+ }
+
+    statusArr = this.state.statuses.map(assessment => {
+      return assessment.status_id
+    })
```
Comment (bryceknz) [inline client/components/ProgressBar.jsx]: There is no need for this object to be created as a class property. We can shift it into the render function. Or better yet - map directly in the return statement!

File: client/components/ProgressBar.jsx [hunk 3]
```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+// import { statement } from '@babel/template'
```
Comment (bryceknz) [inline client/components/ProgressBar.jsx]: Try not to include commented out code in your commits. If it's not being used, there's probably no need for it to be here :-)

File: client/components/App.jsx [hunk 4]
```diff
@@ -10,6 +11,7 @@ const App = () => {
       <Switch>
         <Route path='/register' component={Register} />
         <Route path ='/' component={Header} />
+        <Route path='/progressbar' component={ProgressBar} />
         <h1>MOA development has begun!</h1>
       </Switch>
     </Router>
```
Comment (emilyparkes) [inline client/components/App.jsx]: My bad this shouldn't be called from app when you submit this PR that's just so you can check it renders. Please remove changes to app.

File: tests/client/components/progressBar.test.js (hunk 1 above)
Comment (Bradleywh1) [reply tests/client/components/progressBar.test.js]: Hey Bryce!
This figure ('2341') is the expected array output from mapping over the mock data

File: client/components/ProgressBar.jsx (hunk 2 above)
Comment (Bradleywh1) [reply client/components/ProgressBar.jsx]: Now mapping directly in the return statement

File: client/components/ProgressBar.jsx (hunk 3 above)
Comment (Bradleywh1) [reply client/components/ProgressBar.jsx]: Comment removed

**Review by emilyparkes — APPROVED** [review]

**Pull request merged**